### PR TITLE
🧹 add missing tags to chef client and server policies

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: community/chef-infra-client.mql.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Tim Smith
         email: tim@mondoo.com
+    tags:
+      mondoo.com/category: best-practices
+      mondoo.com/platform: linux
```
Comment (tas50) [inline community/chef-infra-client.mql.yaml:10]: This could be any *nix including macOS or FreeBSD

File: community/chef-infra-client.mql.yaml [hunk 2]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Tim Smith
         email: tim@mondoo.com
+    tags:
+      mondoo.com/category: best-practices
```
Comment (tas50) [inline community/chef-infra-client.mql.yaml:9]: These are both security

**Comment by chris-rock** [comment]
@tas50 updated

File: community/chef-infra-server.mql.yaml [hunk 3]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Tim Smith
         email: tim@mondoo.com
+    tags:
+      mondoo.com/category: security
+      mondoo.com/platform: linux,unix
```
Comment (tas50) [inline community/chef-infra-server.mql.yaml:10]: This one in linux only though. Sorry

Comment (chris-rock) [reply community/chef-infra-server.mql.yaml:10]: Then the asset filter is wrong too

Comment (chris-rock) [reply community/chef-infra-server.mql.yaml]: Updated the tag and also fixed the asset filter

**Review by tas50 — APPROVED** [review]

**Pull request merged**